<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndWidget</class>
 <widget class="QWidget" name="EndWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EndWidget</string>
  </property>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>結束游戲</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toAppInfo">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>資訊頁</string>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>結果：n / k: p%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="replay">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>580</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重新開始</string>
   </property>
  </widget>
  <widget class="QLabel" name="rankPic">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/Administrator/drawables/SRank.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="timeDisplay">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>425</y>
     <width>420</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>總用時：a分b秒</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
